app/representers/api/auth/story_representer.rb

```diff
@@ -18,4 +18,17 @@ def self_url(r)
       href: unpublish_api_authorization_story_path(represented)
     } if !represented.published_at.nil?
   end
+
+  link :audio do
+    {
+      href: api_authorization_audio_files_path(represented.id),
+      count: represented.default_audio.count
+    } if represented.id
+  end
+  embed :default_audio,
```

Comment:
A bit off-topic... but are we assuming that `default_audio` will always be the podcast version?

I can imagine at some point we'll want a link to auth-story-versions in here.  A task for another day!